extensions/amp-ad-network-adsense-impl/0.1/test/test-amp-ad-network-adsense-impl.js

```diff
@@ -360,6 +370,40 @@ describes.sandboxed('amp-ad-network-adsense-impl', {}, () => {
         expect(ampAnalyticsElement.innerHTML.indexOf(url)).to.not.equal(-1);
       });
     });
+    it('centers iframe in slot when height && width', () => {
+      const centerCreativeSpy = sandbox.spy(impl, 'centerCreative_');
+      impl.onCreativeRender(false);
+      expect(centerCreativeSpy).to.be.calledOnce;
+    });
+    it('centers iframe in slot when !height && !width', () => {
+      return createImplTag({
```

Comment:
Sheer paranoia: You're now doing `createImplTag` twice: once in the `beforeEach` and once here.  Are you sure that the  test evaluation is with respect to the right one?